AlphaWallet/Common/Types/Configuration.swift

```diff
@@ -233,7 +233,7 @@ struct Configuration {
             }
 
             static let textFieldIcon = UIColor { trait in
-                return colorFrom(trait: trait, lightColor: R.color.alabaster()!, darkColor: R.color.venus()!)
+                return colorFrom(trait: trait, lightColor: R.color.venus()!, darkColor: R.color.alabaster()!)
```

Comment:
Why switch it around?

Reply:
It was a mistake in the first place. If not switched around, it would be the same colour as the text field background.

Reply:
Not sure how this is used. Is this relevant to this PR? or should it be a separate one? If it's a separate one, can you make a before and after screenshot to show what this does?

Reply:
Before switching | After switching
-|-
![before](https://user-images.githubusercontent.com/1050309/197733620-cb593c58-015f-4c1e-a9f2-cb3f2a9aa425.png)|![after](https://user-images.githubusercontent.com/1050309/197733606-dc314f7e-4da6-4d23-8306-2538c5d36cf9.png)